app/forms/catalog_record_id_form.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require "reform/form/coercion"
+
+class CatalogRecordIdForm < Reform::Form
+  ### classes that define a virtual catalog record ID model object and data structure, used in form editing...persistence is in the cocina model
+  class Row
+    attr_accessor :value, :refresh, :catalog, :_destroy
+
+    def initialize(attrs = {})
+      @value = attrs[:value]
+      @refresh = attrs[:refresh]
+      @catalog = attrs[:catalog]
```

Comment:
Added this to help persist Folio catalog links when in Symphony mode, and vice versa.

Reply:
Sounds good, done.